Buildings/Experimental/RadiantControl/SlabTempSignal/Validation/DeaCon.mo

```diff
@@ -0,0 +1,117 @@
+within Buildings.Experimental.RadiantControl.SlabTempSignal.Validation;
+model DeaCon "Validation model for deadband control"

```

Comment:
Model names should have full words, camel case, with each beginning letter in upper case.